src/CMakeLists.txt

```diff
@@ -62,12 +63,13 @@ if (MSVC)
 		LINK_FLAGS "/subsystem:windows /ENTRY:mainCRTStartup")
 elseif(APPLE)
 	# Help CPack build the bundle
+	set_source_files_properties(${PROJECT_SOURCE_DIR}/asset/brd.icns
+		PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
```

Comment:
Looks like it is set there, but I didn't find any kind or resource in the bundle.